contracts/drafts/meta-tx/Context.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.0;
+
+/*
+ * @dev Provides information about the current execution context, including the
+ * sender of the transaction and its data. While these are generally available
+ * via msg.sender and msg.data, they not should not be accessed in such a direct
+ * manner, since when dealing with meta-transactions the account sending and
+ * paying for execution may not be the actual sender (as far as an application
+ * is concerned).
+ */
+contract Context {
```

Comment:
I would add an empty `constructor` with `internal` modifier to be clear this is an internal contract.